inst/sql/sql_server/field_cdm_field.sql

```diff
@@ -18,7 +18,7 @@ FROM
   select num_violated_rows from
   (
     select
-      case when count_big("@cdmFieldName") = 0 then 0
+      case when count_big(@cdmFieldName) = 0 then 0
```

Comment:
This allows the RDBMS to  naturally interpret the field casing rather than with an explicit casing.